### PR TITLE
Doscp 14337

### PR DESCRIPTION
**Pull request by biniona-mongodb**
## Pull Request Info
Update Node 4.0 Authentication Parameters to specify that `gssapiServiceName` is deprecated. 

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-14337

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/a43f746/node/docsworker-xlarge/DOSCP-14337/fundamentals/authentication/enterprise-mechanisms/

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging links in the PR description updated?

### CODE REVIEW

File: source/fundamentals/authentication/enterprise-mechanisms.txt [hunk 1]
```diff
@@ -21,10 +21,19 @@ parameters of the
...
   parameter if a custom service realm is required.
 
+.. note::
+
+   The ``gssapiServiceName`` parameter, although still supported by the MongoDB Node driver,
+   is a *deprecated* alias for ``authMechanismProperties=SERVICE_NAME:<your service name>``.
+   See the
+   :manual:`authMechanismProperties </reference/connection-string/#urioption.authMechanismProperties>`
+   parameter documentation for more information.
+
```
Comment (terakilobyte) [inline source/fundamentals/authentication/enterprise-mechanisms.txt:36]:
```suggestion
.. important::

   The ``gssapiServiceName`` parameter is deprecated and may be removed in future versions of the driver. Use ``authMechanismProperties=SERVICE_NAME:<your service name>`` in the connection URI instead.
   See the
   :manual:`authMechanismProperties </reference/connection-string/#urioption.authMechanismProperties>`
   parameter documentation for more information.

```

Comment (terakilobyte) [reply source/fundamentals/authentication/enterprise-mechanisms.txt:36]: One suggestion to tighten this up and make it more active and prescriptive.

**Review by terakilobyte — CHANGES_REQUESTED** [review]

File: source/fundamentals/authentication/enterprise-mechanisms.txt (hunk 1 above)
Comment (biniona-mongodb) [reply source/fundamentals/authentication/enterprise-mechanisms.txt]: Thanks for the note. I agree that this prescriptive and direct phrasing is more helpful to users in that it tells them which decision to make. Thanks for drawing my attention to the "important" directive as well, have not used one yet and didn't occur to me (but deprecation warnings are important). Will try to keep these edits in mind going forward.

Fix: https://github.com/mongodb/docs-node/pull/147/commits/a43f7461c88ff0e514f0c02f32cc573cfa673c4a

**Review by terakilobyte — APPROVED** [review]
lgtm

**Pull request merged**